### DISCUSSION TITLE
Suggestion: Support more app launching buttons

### DISCUSSION THREAD

**Discussion by toddsay** [PRProd/HA-Firemote#4] [Ideas]
It appears that it's possible to launch other popular streaming apps from those buttons at the bottom of the simulated remote (HBO Max, Showtime, Starz, etc). If so, it would be handy to either add additional buttons for the most popular apps, and/or make those slots configurable. Does that sound possible?

**Comment by PRProd** [comment]
Yes!  100% possible!  This is something that's been on my to-do list for a while now, but I've been placing a higher priority on reported compatibility issues.  Thanks for the "nudge"!  I'm hoping to get there very soon.

**Comment by PRProd** [comment]
[Version 1.5.0](https://github.com/PRProd/HA-Firemote/releases/tag/v1.5.0) is now available!  Some limited app launch configuration abilities have been added using the apps that were mentioned by various community members.

Feel free to use the [release discussion](https://github.com/PRProd/HA-Firemote/discussions/5) to request additional "hard coded" app options.

Eventually, I'll be working on allowing you to enter your own custom app information using YAML.  I'm not there yet, though.
